### PR TITLE
Propagate check message flags to chat messages created by the treat wounds macro

### PR DESCRIPTION
**Pull request by In3luki**

### CODE REVIEW

**Comment by stwlam** [comment]
Is this addressing a particular problem?

**Comment by In3luki** [comment]
I was testing the `healing-received` selector and noticed that it doesn't work with the macro rolls because they have no flags.

File: src/scripts/macros/treat-wounds.ts [hunk 1]
```diff
@@ -156,6 +156,7 @@ async function treat(
 
             if (riskySurgery) {
                 ChatMessagePF2e.create({
+                    flags: message.flags,
```
Comment (stwlam) [inline src/scripts/macros/treat-wounds.ts:159]: Is this a message document or source? We'd want the source flags.

Comment (In3luki) [reply src/scripts/macros/treat-wounds.ts]: It's the source now.

**Pull request merged**